app/uk/gov/hmrc/hmrcemailrenderer/templates/tcs/newTcsRenewalConfirmationMessageAlert.scala.html

```diff
@@ -1,24 +1,106 @@
-@(params: Map[String, Any])@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params,"Your tax credits renewal has been received", true, footerMessage = None, true) {
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Your tax credits renewal has been received", true, footerMessage = None, true) {
 
```

Comment:
As per contributing guidelines (point 3 -> https://github.com/hmrc/hmrc-email-renderer/blob/master/CONTRIBUTING.md#content-recommendations) here there should be a salutation